docker/ubuntu-indexer/bin/system-setup.sh

```diff
@@ -23,7 +23,9 @@ apt_packages+=('libclang-dev')
 apt_packages+=('llvm')
 apt_packages+=('llvm-dev')
 apt_packages+=('npm')
-apt_packages+=('nodejs')
+
+# Dependency for nodejs
+apt_packages+=('apt-transport-https')
```

Comment:
Yes. I misinterpreted your commit message as curl not previously having been installed at all. Sorry, the fact that apt-transport-https is a move rather than an addition wasn't clear on GH's diff view.